index.bs

```diff
@@ -231,6 +231,43 @@ scenarios. Additional scenarios, including sample code, are given later in [[#sa
     * Web page shows that the selected user is signed in, and navigates to the signed-in page.
 
 
+### New device enrollment ### {#usecase-new-device-enrollment}
+
+This use case illustrates how a [=roaming authenticator=] - e.g., a USB security key fob - can be used for initial sign-in on a
+new device, and a [=platform authenticator=] - typically built into a laptop or smartphone - can be employed for easy day-to-day
+use.
+
+- First, on a desktop computer:
+    * User navigates to example.com in a browser and signs in to an existing account using whatever method they have been using
+        (possibly a legacy method such as a password), or creates a new account.
+    * User navigates to account security settings and selects "Register security key".
+    * Website prompts the user to plug in a USB security key fob and enter a nickname for the key.
+    * User enters "Keychain key" and plugs in the USB key.
+    * The USB key blinks to indicate the user should press the button on it.
+    * User presses the button on the USB key.
+    * Website shows message, "Registration complete."
+
+- Later, on their laptop:
+    * User navigates to example.com in a browser and initiates signing in.
+    * Website prompts the user to plug in the "Keychain key" USB key.
+    * User plugs in the previously registered USB key and presses the button.
+    * Website shows that the user is signed in, and navigates to the signed-in page.
+    * Website prompts, "Do you want to register this device with example.com?"
+    * User agrees.
+    * Laptop prompts the user for a previously configured [=authorization gesture=] (PIN, biometric, etc.); the user
+        provides this.
+    * Website prompts the user for a device nickname; user enters "Work laptop".
+    * Website shows message, "Registration complete."
+    * User signs out.
+
+- Later, again on their laptop:
+    * User navigates to example.com in a browser and initiates signing in.
+    * Website shows message, "If this device is registered, please follow the instructions in your browser. If not, please plug in
```

Comment:
Most likely, yeah, but since cookies and such may be cleared from time to time it's not entirely reliable. But I agree that assuming the common case here makes the use case example more straightforward.